.github/workflows/bats.yaml

```diff
@@ -158,28 +199,45 @@ jobs:
         BATS_COMMAND: >-
           exec xvfb-run --auto-servernum
           --server-args='-screen 0 1280x960x24'
+    - name: "Windows: Set startup command"
+      if: runner.os == 'Windows'
+      shell: bash
+      run: echo "BATS_COMMAND=$BATS_COMMAND" >> "$GITHUB_ENV"
+      env:
+        BATS_COMMAND: wsl.exe -d Debian --exec
 
     - name: Run BATS
       # We use ${{ env.BATS_COMMAND }} instead of ${BATS_COMMAND} to let the
       # shell parse the command, instead of doing it via expansion which is then
-      # parsed differently (--server-args isn't kept as one word).
+      # parsed differently (--server-args isn't kept as one word).  Also, we
+      # need to use the env.* form because PowerShell uses ${ENV:VAR} instead.
       run: >-
         ${{ env.BATS_COMMAND }}
         ./bats-core/bin/bats
-        --gather-test-outputs-in "${LOGS_DIR}"
-        --formatter tap
+        --gather-test-outputs-in '${{ env.LOGS_DIR }}'
+        --formatter cat
         --report-formatter tap
         'tests/${{ steps.normalize.outputs.test }}'
       env:
-        RD_CONTAINER_ENGINE:     ${{ matrix.engine }}
+        BATS_COMMAND:            ${{ env.BATS_COMMAND }}
+        GITHUB_TOKEN:            ${{ github.token }}
+        LOGS_DIR:                ${{ env.LOGS_DIR }}
         RD_CAPTURE_LOGS:         "true"
+        RD_CONTAINER_ENGINE:     ${{ matrix.engine }}
         RD_TAKE_SCREENSHOTS:     "true"
         RD_TRACE:                "true"
         RD_USE_GHCR_IMAGES:      "true"
         RD_USE_RAMDISK:          "true"
-        LOGS_DIR:                ${{ github.workspace }}/logs
-        GITHUB_TOKEN:            ${{ github.token }}
-        BATS_COMMAND:            ${{ env.BATS_COMMAND }}
+        RD_USE_WINDOWS_EXE:      "${{ runner.os == 'Windows' }}"
+        WSLENV: >-
+          :GITHUB_TOKEN:
+          :RD_CAPTURE_LOGS:
+          :RD_CONTAINER_ENGINE:
+          :RD_TAKE_SCREENSHOTS:
+          :RD_TRACE:
+          :RD_USE_GHCR_IMAGES:
+          :RD_USE_RAMDISK:
+          :RD_USE_WINDOWS_EXE:
```

Comment:
Because this is `>-` for YAML, new lines are replaced with spaces; however, we _don't_ want spaces there (otherwise it'll try to export ` RD_TRACE` with a space instead of `RD_TRACE`).  So this just makes WSL export extra environment variables where the name is a single space (which it can't find and silently ignores).

Reply:
Maybe add a comment about that then?

Reply:
Alternatively:

```yaml
       WSLENV: "\
          GITHUB_TOKEN:\
          RD_CAPTURE_LOGS:\
          RD_CONTAINER_ENGINE:\
          RD_TAKE_SCREENSHOTS:\
          RD_TRACE:\
          RD_USE_GHCR_IMAGES:\
          RD_USE_RAMDISK:\
          RD_USE_WINDOWS_EXE:\
          "
```

Or is that too ugly?